templates/createDropbox.hbs

```diff
@@ -59,73 +59,40 @@
-        <input type="text" class="form-control" id="customerName" name="customerName"
-          value="{{ dropbox.customer.name }}" required />
-      </div>
-      <div class="form-group">
-        <label for="customerEmail">Email</label>
-        <input type="text" class="form-control" id="customerEmail" name="customerEmail"
-          value="{{ dropbox.customer.email }}" />
-      </div>
-      <div class="form-group">
-        <label for="customerPhone">Phone number</label>
-        <input type="text" class="form-control" id="customerPhone" name="customerPhone"
-          value="{{ dropbox.customer.phone }}" />
-      </div>
-      <p class='top-border'>
-        Please fill out one of the following so we can identify you.
-      </p>
-      <div class="identifiable-fields">
-        <div class="form-group">
-          <label for="customerDob">Date of birth</label>
-          <input type="date" class="form-control" id="customerDob" name="customerDob"
-            value="{{ dropbox.customer.dob }}" />
-        </div>
-        <div class="form-group">
-          <label for="customerReference">Benefits claim reference number</label>
-          <input type="text" class="form-control" id="customerReference" name="customerReference"
-            value="{{ dropbox.customer.reference }}" />
-        </div>
-        <div class="form-group">
-          <label for="customerNationalInsurance">National insurance number</label>
-          <input type="text" class="form-control" id="customerNationalInsurance" name="customerNationalInsurance"
-            value="{{ dropbox.customer.nationalInsurance }}" />
-        </div>
-        <div class="form-group has-error visually-hidden" id='identifiable-fields-warning'>
-          <p>Please fill out one of the above</p>
-        </div>
-      </div>
-      <div class="form-group top-border">
-        <label for="description">Why are you uploading these documents?</label>
-        <textarea class="form-control" id="description" name="description" rows="3"
-          placeholder='Give a reason for uploading the documents, eg. "to apply for change of circumstance"'
-          required>{{ dropbox.description }}</textarea>
-      </div>
-      <div class="form-group">
-        <button type="submit" id="submitDropbox" class="btn btn-block btn-lg btn-primary">Submit Documents</button>
-      </div>
-    </form>
-    <script type="text/javascript">
-      function validateIdentification() {
-        if (document.querySelector('#customerDob').value !== ''
-          || document.querySelector('#customerReference').value !== ''
-          || document.querySelector('#customerNationalInsurance').value !== '') {
-          return true;
-        } else {
-          document.querySelector('#identifiable-fields-warning').classList.remove('visually-hidden')
-          return false;
-        }
-      }
-    </script>
-  </section>
+<section>
+  <h2>2. Add your details:</h2>
+  <p>
```

Comment:
I don't think this paragraph is relevant anymore

Reply:
ah, good shout!